script/build

```diff
@@ -14,26 +21,28 @@ if [ -n "$CF_KV_TOKEN" ]; then
   fi
 
   if [ "$BRANCH" = "main" ]; then
-    curl -sSLf "$CF_KV_URL" \
+    data=$(curl -sSLf "$CF_KV_URL" \
       -H "Authorization: Bearer $CF_KV_TOKEN" \
-      -H "Content-Type:application/json" | jq . > ./site/public/data.json
-    curl -sSLf https://www.home-assistant.io/integrations.json | jq . > ./site/public/integration_details.json
+      -H "Content-Type:application/json")
+    integration_details=$(curl -sSLf https://www.home-assistant.io/integrations.json)
   else
-    curl -sSLf "$CF_KV_URL_DEV" \
+    data=$(curl -sSLf "$CF_KV_URL_DEV" \
       -H "Authorization: Bearer $CF_KV_TOKEN" \
-      -H "Content-Type:application/json" | jq . > ./site/public/data.json
-    curl -sSLf https://rc.home-assistant.io/integrations.json | jq . > ./site/public/integration_details.json
+      -H "Content-Type:application/json")
+    integration_details=$(curl -sSLf https://rc.home-assistant.io/integrations.json)
   fi
 
 else
-  curl -sSLf https://dev--home-assistant-analytics.netlify.app/data.json | jq . > ./site/public/data.json
-  curl -sSLf https://rc.home-assistant.io/integrations.json | jq . > ./site/public/integration_details.json
+  data=$(curl -sSLf https://dev--home-assistant-analytics.netlify.app/data.json)
+  integration_details=$(curl -sSLf https://rc.home-assistant.io/integrations.json)
 fi
 
+# Export to files
+jq -n "$integration_details" --argjson integration_details "$integration_details" > "$integration_details_path"
```

Comment:
Weird we need to pass data twice. Btw what if the Json contains double quotes, that won’t interfere with the double quotes that wrap the variable?

Reply:
Can do `echo "$var" | jq . > path` if you prefer